README.md

```diff
@@ -76,22 +75,18 @@ $socket = new SecureServer($socket, $loop, array(
 $http = new React\Http\Server($socket);
 ```
 
-For each incoming connection, it emits a `request` event with the respective
+For each incoming connection, it executes the callback function with the respective
 [`Request`](#request) and [`Response`](#response) objects:
```

Comment:
Fixed the missing parts. Also tried to make it inline with the `Server` class. Have a look.